pkg/networkservice/common/localbypass/server.go

```diff
@@ -48,34 +42,23 @@ type localBypassServer struct {
 //                        while maintaining the NewServer pattern for use like anything else in a chain.
 //                        The value in *server must be included in the registry.NetworkServiceRegistryServer listening
 //                        so it can capture the registrations.
-func NewServer(registryServer *registry.NetworkServiceRegistryServer) networkservice.NetworkServiceServer {
-	rv := &localBypassServer{}
-	*registryServer = localbypass.NewNetworkServiceRegistryServer(rv)
+func NewServer(reg localbypass.Registry) networkservice.NetworkServiceServer {
```

Comment:
Why are we passing the registry *in* here when before we were passing it *out* ?